docs/fleet/provision.md

```diff
@@ -164,16 +164,54 @@ This provisioning functionality is provided by the [Viam Agent provisioning subs
 
 If you specify a WiFi network to connect to in your configuration file, the Viam Agent will automatically connect to that network when in range.
 
-You can configure one or more WiFi networks to connect to in the `agent` configuration object in your `viam-server` configuration file.
-For example, to configure SSIDs and passwords for two WiFi networks named `primaryNet` and `fallbackNet`, you can use the following configuration:
+To configure the Agent with the necessary WiFi connection information and credentials, add an `agent` configuration object to your `viam-server` configuration file that contains this information.
+
+For example, the following configuration defines the connection information and credentials necessary to connect to a `exampleNet` network:
 
 ```json {class="line-numbers linkable-line-numbers"}
 ...
 "agent": {
   "agent-provisioning": {
-    ...
+    "release_channel": "stable",
     "attributes": {
-      ...
+      "networks": [
+        {
+          "type": "wifi",
+          "ssid": "exampleNet",
+          "psk": "myFirstPassword"
+        },
+      ]
+    }
+  }
+}
+```
+
+You can add this configuration to the <file>/etc/viam.json</file> configuration file you deploy to your machine, or from the **CONFIGURE** tab in the [Viam app](https://app.viam.com/) for your machine, using **Raw JSON** mode.
+
+The Viam Agent will connect to and use the WiFi network you specify even if that network does not have internet access.
```

Comment:
This is true ONLY for the "primary" network. Most always, that's the one configured during provisioning, not here in the config. Otherwise, it'll be whatever network has the highest priority. (A network configured via provisioning in single mode will have priority 999.) When not providing priorities, here, they'll default to 0 priority. (Range is -999 to + 999)